sys-libs/pam/pam-1.5.2-r1.ebuild

```diff
@@ -62,10 +62,10 @@ multilib_src_configure() {
 	local myconf=(
 		CC_FOR_BUILD="$(tc-getBUILD_CC)"
 		--with-db-uniquename=-$(db_findver sys-libs/db)
-		--with-xml-catalog=/etc/xml/catalog
-		--enable-securedir=/$(get_libdir)/security
-		--includedir=/usr/include/security
-		--libdir=/usr/$(get_libdir)
+		--with-xml-catalog="${EPREFIX}"/etc/xml/catalog
+		--enable-securedir="${EPREFIX}"/$(get_libdir)/security
+		--includedir="${EPREFIX}"/usr/include/security
+		--libdir="${EPREFIX}"/usr/$(get_libdir)
```

Comment:
In commit e4fee95b96c9d684c3b992f7e154c9d12e6636ab `--exec-prefix="${EPREFIX}"` is also added here. I wonder if that's still needed or not.

Reply:
> @mikhailkoliada See https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=e4fee95b96c9d684c3b992f7e154c9d12e6636ab, which you signed off on. I don't know why prefix changes have been reverted in the latest version. See also bug https://bugs.gentoo.org/695966 (and other bugs linked in the commit above) if you need a reminder of previous discussions. It's been a pain to avoid pulling pam in prefix, so it's better to just add support and move on, it doesn't hurt having it installed.

Hi @amadio! I believe that I discussed it with grobian back then to drop all the pam and mask pam USE on prefix, does it make sense to build it now though? Or only as a matter of deptree resolution?

Reply:
It doesn't make much sense, it's just less painful to let it be installed than trying to hunt down all ways in which it can be pulled, but of course, if now we can manage to avoid pulling it, it would be a better solution.

Reply:
> In commit [e4fee95](https://github.com/gentoo/gentoo/commit/e4fee95b96c9d684c3b992f7e154c9d12e6636ab) `--exec-prefix="${EPREFIX}"` is also added here. I wonder if that's still needed or not.

[Logs](https://raw.githubusercontent.com/wiredhikari/prefix_on_riscv/main/logs/stage3_pam/stage3.log)  for the pam ebuild according to this pr.

Reply:
There's double prefix in a lot of places in your logfile...

Reply:
I forgot to point it out, all the attempts of installing prefix get added in the log file and several failed installations of `sys-libs/pam` can be found. The successful pam install according to ebuild in this pr starts from line 51816 in the log file, I have trimmed the important logs [here](https://raw.githubusercontent.com/wiredhikari/prefix_on_riscv/main/logs/stage3_pam/stage3_working_pam.log).